tests/e2e/specs/shopper/cart-checkout/account.test.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../../utils';
+import { SIMPLE_PHYSICAL_PRODUCT_NAME } from '.../../../../utils/constants';
+
+/**
+ * External dependencies
+ */
+import {
+	visitBlockPage,
+	selectBlockByName,
+	saveOrPublish,
+} from '@woocommerce/blocks-test-utils';
+import {
+	merchant,
+	setCheckbox,
+	openDocumentSettingsSidebar,
+} from '@woocommerce/e2e-utils';
+import { switchUserToAdmin, visitAdminPage } from '@wordpress/e2e-test-utils';
+
+// Defining block details
+const block = {
+	name: 'Checkout',
+	slug: 'woocommerce/checkout',
+	class: '.wp-block-woocommerce-checkout',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 ) {
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'Skipping Cart & Checkout tests', () => {} );
```

Comment:
To a developer new to the team, would it be obvious why we're choosing to skip tests here?